src/Publication/PublicationRequestFilters.js

```diff
@@ -26,53 +27,27 @@ export class PublicationRequestFilters {
   }
 
   sanitize(req) {
+    let by = server.extractFromReq(this.req, 'sort_by')
```

Comment:
`server.extractFromReq` throws if the key is not present.
Maybe we should wrap this in a `trycatch`? I'll leave it up to you, I think it might not be a bad idea to throw here

Reply:
I removed DEFAULT_VALUES so it will throw (the UI doesn't hit that endpoint without all the params tho)